### PR TITLE
[JR-190] Fill more info panel

### PR DESCRIPTION
**Pull request by E1379**

### CODE REVIEW

File: src/views/MediaChannels/components/MoreInfoDrawer.tsx [hunk 1]
```diff
@@ -19,37 +91,212 @@ interface MoreInfoDrawerProps {
...
+                href={intl.formatMessage(messages.learnWhyLink)}
+                target="_blank"
+              >
+                <FormattedMessage {...messages.learnWhy} />
+              </Link>
+            </span>
+          </Typography>
+        )}
+
+        {Array.isArray(cross_postings) && cross_postings.length > 0 && (
```
Comment (berthj193) [inline src/views/MediaChannels/components/MoreInfoDrawer.tsx:208]: Isn't this covered by `product` type?

File: src/views/MediaChannels/components/MoreInfoDrawer.tsx [hunk 2]
```diff
@@ -19,37 +91,212 @@ interface MoreInfoDrawerProps {
   onClose: () => void;
 }
 
+function isValidHttpUrl(string) {
```
Comment (berthj193) [inline src/views/MediaChannels/components/MoreInfoDrawer.tsx]: Would it better to move this to utilities?

**Review by berthj193 — APPROVED** [review]
Code looks good to me. 🚀

File: src/views/MediaChannels/components/MoreInfoDrawer.tsx (hunk 1 above)
Comment (E1379) [reply src/views/MediaChannels/components/MoreInfoDrawer.tsx:208]: Not really, `product` is data fetch from external source at runtime. TS does not do runtime type validation unfortunately.

File: src/views/MediaChannels/components/MoreInfoDrawer.tsx (hunk 2 above)
Comment (E1379) [reply src/views/MediaChannels/components/MoreInfoDrawer.tsx]: Yes, you're right. Moved!

File: src/views/MediaChannels/components/MoreInfoDrawer.tsx [hunk 3]
```diff
@@ -1,15 +1,88 @@
...
+    },
   },
-  moreInfologo: {
+  close: {
+    position: "absolute",
+    textAlign: "end",
+    top: 0,
     width: "100%",
+    background:
+      "linear-gradient(180deg, white 0%, white 75%, transparent 100%)",
```
Comment (marianmazarovici) [inline src/views/MediaChannels/components/MoreInfoDrawer.tsx]: is this (and other similar styles) theme-specific? shouldn't it be extracted?

Comment (E1379) [reply src/views/MediaChannels/components/MoreInfoDrawer.tsx]: Only if the background will not be white. But for now I just removed the gradient. That way it is always correct.

**Review by marianmazarovici — APPROVED** [review]

**Pull request merged**